Gruntfile.js

```diff
@@ -223,7 +223,7 @@ module.exports = function(grunt) {
           testname: 'SVGTypewriter Sauce Unit Tests',
           browsers: [{
             browserName: "firefox",
-            version: "30" 
+            version: "30"
```

Comment:
We may want to not specify the version number for both FF and chrome, such that it is always the latest on SauceLabs